src/main/java/com/dubture/jenkins/digitalocean/ComputerLauncher.java

```diff
@@ -128,8 +113,16 @@ public void onClosed(Channel channel, IOException cause) {
 
             successful = true;
         } catch (Exception e) {
+            try {
+                Jenkins.getInstance().removeNode(computer.getNode());
+            } catch (Exception ee) {
+                ee.printStackTrace(logger);
```

Comment:
If you look a few lines below, that's what the original code was doing before there was only one exception handing in there

```Java
e.printStackTrace(logger);
```

so I just reused it for the nested exception handling too.